DataFormats/Scouting/interface/Run3ScoutingPFJet.h

```diff
@@ -0,0 +1,136 @@
+#ifndef DataFormats_Run3ScoutingPFJet_h
+#define DataFormats_Run3ScoutingPFJet_h
+
+#include <vector>
+
+//class for holding PF jet information, for use in data scouting
+//IMPORTANT: the content of this class should be changed only in backwards compatible ways!
+class Run3ScoutingPFJet {
+public:
+  //constructor with values for all data fields
+  Run3ScoutingPFJet(float pt,
+                    float eta,
+                    float phi,
+                    float m,
+                    float jetArea,
+                    float chargedHadronEnergy,
+                    float neutralHadronEnergy,
+                    float photonEnergy,
+                    float electronEnergy,
+                    float muonEnergy,
+                    float HFHadronEnergy,
+                    float HFEMEnergy,
+                    int chargedHadronMultiplicity,
+                    int neutralHadronMultiplicity,
+                    int photonMultiplicity,
+                    int electronMultiplicity,
+                    int muonMultiplicity,
+                    int HFHadronMultiplicity,
+                    int HFEMMultiplicity,
+                    float HOEnergy,
+                    float csv,
+                    float mvaDiscriminator,
+                    std::vector<int> constituents)
+      : pt_(pt),
+        eta_(eta),
+        phi_(phi),
+        m_(m),
+        jetArea_(jetArea),
+        chargedHadronEnergy_(chargedHadronEnergy),
+        neutralHadronEnergy_(neutralHadronEnergy),
+        photonEnergy_(photonEnergy),
+        electronEnergy_(electronEnergy),
+        muonEnergy_(muonEnergy),
+        HFHadronEnergy_(HFHadronEnergy),
+        HFEMEnergy_(HFEMEnergy),
+        chargedHadronMultiplicity_(chargedHadronMultiplicity),
+        neutralHadronMultiplicity_(neutralHadronMultiplicity),
+        photonMultiplicity_(photonMultiplicity),
+        electronMultiplicity_(electronMultiplicity),
+        muonMultiplicity_(muonMultiplicity),
+        HFHadronMultiplicity_(HFHadronMultiplicity),
+        HFEMMultiplicity_(HFEMMultiplicity),
+        HOEnergy_(HOEnergy),
+        csv_(csv),
+        mvaDiscriminator_(mvaDiscriminator),
+        constituents_(constituents) {}
```

Comment:
This `vector` could benefit from moving
```suggestion
        constituents_(std:move(constituents)) {}
```